linux/xkb/symbols/3l

```diff
@@ -38,6 +38,8 @@ xkb_symbols "basic" {
     key <AD08> { [ c,         C,         greater,      Greek_chi,      2,         B,          U2A7E,    Greek_CHI ] };
     key <AD09> { [ w,         W,         equal,        Greek_omega,    3,         C,          U2261,    Greek_OMEGA ] };
     key <AD10> { [ b,         B,         ampersand,    Greek_beta,     NoSymbol,  NoSymbol,   U2248,    Greek_BETA ] };
+    key <AD11> { [ Super_R    Super_R    Super_R       Super_R         Super_R    Super_R     Super_R   Super_R  ] };
```

Comment:
I think it would be better to keep these keys undefined, and allow users to set them, probably thru xmodmap.

Reply:
i never meant to add that commit to the pull request. i opened the pull request  before these commits just for the colon. i didnt realize that it would be added automatically.

Reply:
all fixed

Reply:
users can set keys through xmodmap even if they are defined